hack/generator/pkg/codegen/storage/storage_type_factory.go

```diff
@@ -63,7 +67,10 @@ func (f *StorageTypeFactory) Types() (astmodel.Types, error) {
 
 func (f *StorageTypeFactory) process() error {
```

Comment:
Some of the steps of processing require access to `outputTypes` and we can't pass it through because the method signatures are constrained. If I made `outputTypes` a local that was returned, some of the transformations wouldn't work. I'll add details about this into comments.